[2022-06-30 17:22:55,417] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_adjusted.local_to_gcs_task manual__2022-06-30T17:22:48.431558+00:00 [queued]>
[2022-06-30 17:22:55,429] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_adjusted.local_to_gcs_task manual__2022-06-30T17:22:48.431558+00:00 [queued]>
[2022-06-30 17:22:55,430] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-06-30 17:22:55,430] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-06-30 17:22:55,430] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-06-30 17:22:55,443] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2022-06-30 17:22:48.431558+00:00
[2022-06-30 17:22:55,448] {standard_task_runner.py:52} INFO - Started process 5700 to run task
[2022-06-30 17:22:55,454] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_adjusted', 'local_to_gcs_task', 'manual__2022-06-30T17:22:48.431558+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/data_adjusted.py', '--cfg-path', '/tmp/tmpgljqjybd', '--error-file', '/tmp/tmp6q__ndgr']
[2022-06-30 17:22:55,455] {standard_task_runner.py:77} INFO - Job 28: Subtask local_to_gcs_task
[2022-06-30 17:22:55,517] {logging_mixin.py:109} INFO - Running <TaskInstance: data_adjusted.local_to_gcs_task manual__2022-06-30T17:22:48.431558+00:00 [running]> on host d4c453d82639
[2022-06-30 17:22:55,582] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_adjusted
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2022-06-30T17:22:48.431558+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-30T17:22:48.431558+00:00
[2022-06-30 17:22:56,084] {python.py:175} INFO - Done. Returned value was: None
[2022-06-30 17:22:56,097] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_adjusted, task_id=local_to_gcs_task, execution_date=20220630T172248, start_date=20220630T172255, end_date=20220630T172256
[2022-06-30 17:22:56,148] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-06-30 17:22:56,188] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
